src/aktualizr_secondary/CMakeLists.txt

```diff
@@ -20,6 +20,7 @@ add_library(aktualizr_secondary_static_lib STATIC
     $<TARGET_OBJECTS:storage>
     $<TARGET_OBJECTS:logging>
     $<TARGET_OBJECTS:uptane>
+    $<TARGET_OBJECTS:primary>
```

Comment:
This seems less than ideal. What does the secondary need from the primary?

Reply:
We don't need it really, it is old artifact